### PR TITLE
Newspaper Archive : Checkout Benefit Add, Copy Checks

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

Update:
- Checkout benefit list includes newspaper archive benefit for v1,v2,control (see screenshot below)
- Recheck copy throughout against latest [**Figma**](https://www.figma.com/design/h53HMhA753GLQ4ycN1uoPc/newspaper-archive?node-id=545-17670&t=Lm4eiadk4JMJGR7f-1) updates

image

[**Trello Card**](https://trello.com/c/FjKV8faD/1054-update-copy-and-imagery-newspaper-archive-benefit-to-lp)

## How to test

dev => `https://support.thegulocal.com/...`
prod => `https://support.theguardian.com/...`

`.../uk/contribute#ab-newspaperArchiveBenefit=v1`
`.../uk/contribute#ab-newspaperArchiveBenefit=v2`
`.../uk/contribute#ab-newspaperArchiveBenefit=control`

eg prod
`https://support.theguardian.com/uk/contribute#ab-newspaperArchiveBenefit=v1`

### CODE REVIEW

File: support-frontend/assets/helpers/productCatalog.tsx [hunk 1]
```diff
@@ -344,39 +344,36 @@ export const productCatalogDescription: Record<ProductKey, ProductDescription> =
...
+> = {
+	...productCatalogDescription,
+	TierThree: {
+		...productCatalogDescription.TierThree,
+		benefits: [
+			...productCatalogDescription.TierThree.benefits,
+			{
+				copy: `Unlimited access to the Guardian's 200-year newspaper archive`,
+				isNew: true,
+				tooltip: `Look back on more than 200 years of world history with the Guardian archive. Get digital access to every front page, article and advertisement printed in the newspaper from 1981.`,
```
Comment (tjmw) [inline support-frontend/assets/helpers/productCatalog.tsx:359]: Is `1981` here correct? Elsewhere we mention `1821`.

Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/productCatalog.tsx:359]: Good question, it matches the Figma file change.  Does not seem to align the archive availability with  200yrs to be fair. Will check now.

Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/productCatalog.tsx]: Was 1821, updated

**Review by tjmw — APPROVED** [review]
Looks good 👍

**Pull request merged**